mindmeld/components/_config.py

```diff
@@ -464,11 +464,28 @@
     },
 }
 
+DEFAULT_AUGMENTATION_CONFIG = {
+    "augmentor_class": "EnglishParaphraser",
```

Comment:
Do we have an opinion of if there would be multiple english paraphrasers in the future? If not, can we just use the locale and language to decide which paraphraser to use instead of specifying it here?